packages/manager/apps/web/client/app/index.html

```diff
@@ -41,6 +41,8 @@
                     data-universe-click="$ctrl.openSidebar()"
                 >
                 </ovh-manager-navbar>
+                <ovh-manager-account-sidebar me="$ctrl.me">
```

Comment:
I think the `me` binding was done due to the fact that it was already fetched in the parent component (navbar). Maybe it could make sense to fetch in the account-sidebar

Reply:
yep i forgot but in account-sidebar it's an optional binding (if it's not present it'll be fetched by the sidebar) so i removed the binding in web (other managers /me is fetched so i can pass it to the sidebar but in web /me is modified, some field are snakeCased and it breaks the account-sidebar so i'll let the account-sidebar fetch /me in that case)